androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/TitledSection.kt

```diff
@@ -57,6 +57,51 @@ fun TitledSection(
     }
 }
 
+
+@Composable
+fun TitledSectionWithRettsColor(
+    title: String,
+    modifier: Modifier = Modifier,
+    color: Color = Color.White,
```

Comment:
Maybe we should use the type `rettsColor: RettsColor? = null` so that the view can decide how to paint the color, and possibly also add a text description of the color.